packages/suite/src/actions/wallet/discoveryActions.ts

```diff
@@ -449,7 +449,7 @@ export const start =
 
         const solanaAccountInfo: AccountInfo = {
             descriptor:
-                'zpub6qy39U3XHtW4hyk3JXs5UjThu1JAuJjAStCm8nPr9Bb13yX8nJNfX8hiQr56NC4CGnHfvzkiaaFZ7NRsoaUqYZRhkeCyjJM2qRz7ibDiRGh',
+                'zpub6qy39U3XHtW4hyk3JXs5UjThu1JAuJjAStCm8nPr9Bb13yX8nJNfX8hiQr56NC4CGnHfvzkiaaFZ7NRsoaUqYZRhkeCyjJM2qRz7ibDiRGh(fallback)',
```

Comment:
The way address discovery works is that the `account` is checked for the `addresses` field. The returned address is *either* the first of the `unused` addresses of the account *or* the `descriptor` of the account. ETH based accounts don't have addresses associated with them and just use the descriptor field, BTC adjacent chains use the addresses field.   
We should consult FW guys about which Solana uses, but I'm not sure it even matters, since the code should just work either way.